lib/WorldState.js

```diff
@@ -200,7 +200,9 @@ module.exports = class WorldState {
      * The state of the Void Trader
      * @type {VoidTrader}
      */
-    [this.voidTrader] = parseArray(deps.VoidTrader, data.VoidTraders, deps);
+    const voidTraderArray = parseArray(deps.VoidTrader, data.VoidTraders, deps);
+    voidTraderArray.sort((a, b) => Date.parse(a.activation) - Date.parse(b.activation));
+    this.voidTraders = voidTraderArray;
```

Comment:
```suggestion
    /**
      * The state of all Void Traders
      * @type {VoidTrader[]}
      */
    this.voidTraders = parseArray(deps.VoidTrader, data.VoidTraders, deps)
      .sort((a, b) => Date.parse(a.activation) - Date.parse(b.activation));
    [this.voidTrader] = this.voidTraders;
```

then move the jsdoc onto voidTrader, add `@deprecated`, and make new jsdoc for the array.
you'll probably want to check the linting locally, cause i think my indentation isn't sufficient.